handler/music_handler.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../model/music'
+require_relative '../model/genre'
+require_relative '../storage'
+
+class MusicHandler
+  @storage = Storage.new
+  def self.add_music(app)
+    print 'Is it on spotify? [Yes/No]: '
+    gets.chomp
+    puts 'Music created successfully !!!'
+    @storage.save_music(app)
```

Comment:
It looks like you are saving an empty music list, because you didn't update your music list inside the `app` file.